src/fixate/drivers/pps/helper.py

```diff
@@ -206,3 +198,14 @@ def idn(self):
                 inspect.currentframe().f_code.co_name
             )
         )
+
+    def get_identity(self):
```

Comment:
Implement this by calling idn? or the bind the name as an alias?
self.get_identify = self.idn should work...?

Reply:
PPS was a bit of a mess, bk 178x doesn't seem to even us idn, spd 3303x only mentions it in the api list, which gives me conniptions and I was trying to avoid looking at it. Not how binding names works with inheritance either.